CONTRIBUTING.md

```diff
@@ -16,17 +16,30 @@
 *******************************************************************************/-->
 
 # How to Contribute
-We welcome community contributions to Intel(R) oneAPI Data Analytics Library. If you have an idea how to improve the product, you can:
+We welcome community contributions to Intel(R) oneAPI Data Analytics Library. You can:
 
-* Let us know about your proposal via [Issues on oneDAL GitHub\*](https://github.com/oneapi-src/oneDAL/issues).
-* Contribute your changes directly to the repository through [pull request](#pull-requests). 
+- Submit your changes directly with a [pull request](https://github.com/oneapi-src/oneDAL/pulls).
+- Log a bug or feedback with an [issue](https://github.com/oneapi-src/oneDAL/issues).
+
+Refer to our guidelines on [pull requests](#pull-requests) and [isssues](#issues) before you proceed.
+
+## Issues
+
+Use [GitHub issues](https://github.com/oneapi-src/oneDAL/issues) to:
+- report an issue
+- provide feedback
```

Comment:
I propose to use another channels to get feedbacks.

It's a little bit confusing to request feedbacks as an 'issue'

Reply:
agree, fixed